### PR TITLE
docs: Howto on foreground color on timelines

### LINKED ISSUE #4523: Documentation is missing on how to change foreground colors
### Description

When I use custom colors on a timeline diagram, the foreground color does not always provide enough contrast and then I want to specify that color myself.

It was not clear how to do that.

### Steps to reproduce

1. Check https://mermaid.js.org/syntax/timeline.html#themes
2. Search for foreground

### Screenshots

_No response_

### Code Sample

_No response_

### Setup

_No response_

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by mcbeelen**
## :bookmark_tabs: Summary

Added some info on changing foreground colors with cScaleLabel theme variable.

Resolves #4523

Make sure you

- [X] :book: have read the [contribution guidelines](https://github.com/mermaid-js/mermaid/blob/develop/CONTRIBUTING.md)
- [X] :notebook: have added documentation (if appropriate)
- [X] :bookmark: targeted `develop` branch

### CODE REVIEW

File: docs/syntax/timeline.md
Comment (sidharthv96) [inline docs/syntax/timeline.md:1]: Please make the changes in [/packages/mermaid/src/docs/syntax/timeline.md](https://github.com/mermaid-js/mermaid/blob/3c764197fbd1522d9c1951498b3cf4200558b3e7/packages/mermaid/src/docs/syntax/timeline.md).

Comment (mcbeelen) [reply docs/syntax/timeline.md:1]: I moved the changes to the indicated file.

It did make me wonder: Is the code in `/docs` still being used?
Should I have made the changes in two locations?

Comment (sidharthv96) [reply docs/syntax/timeline.md:1]: Code in `/docs` will be automatically processed and updated when you're committing locally (or by the CI if you have github actions enabled in your fork).

**Comment by sidharthv96** [comment]
The current failure is due to formatting. 

> Running `pnpm run lint` failed. Running `pnpm -w run lint:fix` may fix this issue.  If this error doesn't occur on your local machine, make sure your packages are up-to-date by running `pnpm install`.

**Pull request merged**